rest/models/summaries.go

```diff
@@ -0,0 +1,54 @@
+package models
+
+import "github.com/polygon-io/ptime"
+
+type GetSummaryParams struct {
+	// The ticker list to get summaries for
+	TickerAnyOf *string `query:"ticker.anyOf"`
+}
+
+func (p GetSummaryParams) WithTickerAnyOf(q string) *GetSummaryParams {
+	p.TickerAnyOf = &q
+	return &p
+}
+
+type GetSummaryResponse struct {
+	BaseResponse
+	Results []SummaryResult `json:"results,omitempty"`
+}
+
+type SummaryResult struct {
+	Price        float64  `json:"price,omitempty"`
+	Name         string   `json:"name,omitempty"`
+	Ticker       string   `json:"ticker,omitempty"`
+	Branding     Branding `json:"branding,omitempty"`
+	MarketStatus string   `json:"market_status,omitempty"`
+	Type         string   `json:"type,omitempty"`
+	Session      Session  `json:"session,omitempty"`
+	Option       Option   `json:"option,omitempty"`
+}
+
+type Session struct {
+	Change                    float64 `json:"change,omitempty"`
+	ChangePercent             float64 `json:"change_percent,omitempty"`
+	EarlyTradingChange        float64 `json:"early_trading_change,omitempty"`
+	EarlyTradingChangePercent float64 `json:"early_trading_change_percent,omitempty"`
+	LateTradingChange         float64 `json:"late_trading_change,omitempty"`
+	LateTradingChangePercent  float64 `json:"late_trading_change_percent,omitempty"`
+	Close                     float64 `json:"close,omitempty"`
+	High                      float64 `json:"high,omitempty"`
+	Low                       float64 `json:"low,omitempty"`
+	Open                      float64 `json:"open,omitempty"`
+	PreviousClose             float64 `json:"previous_close,omitempty"`
+	Volume                    float64 `json:"volume,omitempty"`
+}
+
+//easyjson:json
+type Option struct {
+	ContractType      string                       `json:"contract_type,omitempty"`
+	ExerciseStyle     string                       `json:"exercise_style,omitempty"`
+	ExpirationDate    ptime.IDaysPolygonDateString `json:"expiration_date,omitempty"`
```

Comment:
You don't want to use `ptime` here. Currently this repo has no dependency on `github.com/polygon-io/ptime` even though we have other models that represent time similar to `ptime.IDaysPolygonDateString`.

We don't want to unnecessarily introduce new dependencies so go find another example where we have a field similar to `ptime.IDaysPolygonDateString` and see how this codebase solves it.